doc/api/tls.md

```diff
@@ -186,11 +185,17 @@ of an application. The `--tls-cipher-list` switch should by used only if
 absolutely necessary.
 
 ## Class: tls.Server
+<!--
```

Comment:
Could you make sure that the comment always starts with `<!-- YAML` (including the space)?

You can run `make doc-only` and then look at `out/doc/api/tls.html` to check that everything renders as expected